packages/internals/src/utils/isCi.ts

```diff
@@ -1,5 +1,52 @@
-import { isCi as isCiLib } from 'is-ci'
-
+// Returns true if the current environment is a CI environment.
 export const isCi = (): boolean => {
-  return !process.stdin.isTTY || isCiLib || Boolean(process.env.GITHUB_ACTIONS)
+  const env = process.env
+
+  // From https://github.com/watson/ci-info/blob/44e98cebcdf4403f162195fbcf90b1f69fc6e047/index.js#L54-L61
+  // Evaluating at runtime makes it possible to change the values in our tests
+  // This list is probably not exhaustive though `process.env.CI` should be enough
+  //  but since we were using this utility in the past, we want to keep the same behavior
+  return !!(
```

Comment:
Do we need all of these? Maybe let's just leave what GitHub Actions and Buildkite use?

Reply:
Actually I think we can just check for `env.CI`:

* https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
* https://buildkite.com/docs/pipelines/environment-variables#CI

Reply:
The detection is user facing so that users don't get their CI run hanging forever on our interactive prompts.
These are from `ci-info` package which was what the previous logic was using, so there is no expected change after.